tasks/connectors/aqua/lib/aqua_helper.rb

```diff
@@ -5,10 +5,26 @@
 module Kenna
   module Toolkit
     module AquaHelper
-      SAAS_AUTH_URL = "https://api.cloudsploit.com/v2/signin"
-      WP_URL_API    = "https://prov.cloud.aquasec.com/v1/envs"
+      def select_region_urls(aqua_url)
+        case aqua_url
+        when /eu-1\.cloud\.aquasec\.com/
+          @saas_auth_url = "https://eu-1.api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov-eu-1.cloud.aquasec.com/v1/envs"
+        when /asia-1\.cloud\.aquasec\.com/
+          @saas_auth_url = "https://asia-1.api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov-asia-1.cloud.aquasec.com/v1/envs"
+        when /ap-2\.cloud\.aquasec\.com/
+          @saas_auth_url = "https://ap-2.api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov-ap-2.cloud.aquasec.com/v1/envs"
+        else
+          @saas_auth_url = "https://api.cloudsploit.com/v2/signin"
+          @wp_url_api = "https://prov.cloud.aquasec.com/v1/envs"
+        end
```

Comment:
Instead of a case statement, these should be stored in a frozen hash, like:
```ruby
REGION_URLS = {
  "eu-1" => {auth: "https://eu-1.api.cloudsploit.com/v2/signin", wp: "https://prov-eu-1.cloud.aquasec.com/v1/envs"},
  "asia-1" => {auth: "https://asia-1.api.cloudsploit.com/v2/signin", wp: "https://prov-asia-1.cloud.aquasec.com/v1/envs"},
  ...
}.freeze
``` 
Then you can write accessor methods that look up auth and wp URLs based on a saved region (the key).

Reply:
Or if they all follow a pattern, you can build the URL using the `URI` class you're already using to parse the host. Then you don't have to repeat the parts of the string that are in common.